### DISCUSSION TITLE
Answer Key

### DISCUSSION THREAD

**Discussion by ashrth** [Asabeneh/30-Days-Of-JavaScript#400] [Ideas]
Hi Asabeneh, I love you JavaScript course, could you also add an answer key for the questions? It would help a lot.

**Comment by Azkarion** [comment]
i need it too because im really new for program language and i want to learn java

**Comment by Ghosh-95** [comment]
An Answer page for the solution of those exercise would be a great help to us

**Comment by ghost** [comment]
@Asabeneh shall we help you getting this request or you think it's not needed?

**Comment by Toothless64652** [comment]
I love it too, it's easy to learn to other programming language

**Comment by owaishkhan18** [comment]
hii i want to answersheet of day 8
